internal/pkg/utils/errors/writer.go

```diff
@@ -127,48 +123,47 @@ func (w *writer) WriteNestedError(level int, main error, errs []error, trace Sta
 
 	// Convert sub errors to string
 	subErrsWriter := w.clone()
-	subErrsWriter.WriteErrorsList(level, errs)
+	subErrsWriter.WriteErrorsList(level+1, errs)
 	subErrsStr := subErrsWriter.String()
 
 	// If there is more than one error or the message is long,
 	// then break line and create bullet list
 	w.Write(mainStr)
-	if errsCount > 1 || len(mainStr)+len(subErrsStr) > 60 || strings.Contains(subErrsStr, "\n") {
-		w.WriteNewLine()
-		if errsCount == 1 {
-			w.WriteIndent(level)
-			w.WriteBullet()
-			w.WriteErrorLevel(level+1, errs[0], nil)
-		} else {
-			w.WriteErrorsList(level, errs)
-		}
-	} else {
+	if errsCount <= 1 && len(mainStr)+len(subErrsStr) <= 60 && !strings.Contains(subErrsStr, "\n") {
 		w.Write(" ")
-		w.WriteErrorsList(level, errs)
+		w.WriteError(errs[0])
+	} else {
+		w.WriteNewLine()
+		w.WriteErrorsList(level+1, errs)
```

Comment:
2. Ostatne zmeny su fix pre edge case, kde to vygenerovalo `- - error1` namiesto `- error 1`, vid test.
`level` urcuje, ako velmi je chyba vnorena -> ako ma byt odsadena.